### PR TITLE
[wip] - [HttpRepository] add new repository to simplify usage of query bundle in http

### PR DESCRIPTION
**Pull request by sensorario**
| Q             | A
| ------------- | ---
| Branch?       | 2.2
| Minor version?      | (2.2)
| New feature?  | yes 
| BC breaks?    | no
| Deprecations? | no 
| Tests pass?   | yes

 - [ ] unit tests

### CODE REVIEW

File: src/Mado/QueryBundle/Repositories/HttpRepository.php [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    public function buildForEntity(string $entityClassName)
+    {
+        if (!$this->repo) {
+            $this->repo = new BaseRepository(
+                $this->manager,
+                $this->manager->getClassMetadata($entityClassName)
+            );
+        }
+
+        $this->repo->setRequest($this->stack->getCurrentRequest());
```
Comment (sensorario) [inline src/Mado/QueryBundle/Repositories/HttpRepository.php]: @orociscaverai I started to play with QueryBundle few days ago and the result is here. A way to hide `setRequest`.

````php
    $httpRepository->buildForEntity(\App\Entity\Initiatives::class);
    $httpRepository->getResult();
```

May we work together with #170 and #167?

Comment (AlessandroMinoccheri) [reply src/Mado/QueryBundle/Repositories/HttpRepository.php]: @orociscaverai what do you think?

Comment (sensorario) [reply src/Mado/QueryBundle/Repositories/HttpRepository.php]: up

Comment (orociscaverai) [reply src/Mado/QueryBundle/Repositories/HttpRepository.php]: OK

**Comment by AlessandroMinoccheri** [comment]
Ok

**Pull request closed without merging**